sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/CognitiveComplexityWalker.cs

```diff
@@ -31,6 +31,8 @@ public sealed class CognitiveComplexityWalker : CSharpSyntaxWalker
     {
         private readonly List<SecondaryLocation> incrementLocations = new List<SecondaryLocation>();
         private readonly List<ExpressionSyntax> logicalOperationsToIgnore = new List<ExpressionSyntax>();
+        private readonly Queue<BinaryExpressionSyntax> binaryNodesToVisit =

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'binaryNodesToVisit'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1144)